addons/web/static/src/core/barcode/barcode_video_scanner.js

```diff
@@ -157,7 +169,7 @@ export class BarcodeVideoScanner extends Component {
         } catch (err) {
```

Comment:
Not related to this PR, but as I discover the code... The try clause contains way to many things. It should be limited to the instruction that may throw and for which we want to catch the error (in this case, I assume `this.detector.detect`). Because as it is, if there's a programming error, let's say because overlayInfo is undefined at some point for instance, the error will be caught like if it was a "business error".